sections/dashboard/CurrencyConvertCard/SlippageSelector.tsx

```diff
@@ -45,11 +50,19 @@ export const SlippageSelector: FC<SlippageSelectorProps> = () => {
 							<SolidTooltipCustomValue
 								value={customSlippage}
 								onChange={(_, value) => {
-									if (value !== '') {
-										const numValue = Number(value);
-										setSlippage(numValue >= 0 ? numValue : DEFAULT_SLIPPAGE);
+									const numValue = Number(value);
+
+									if (value === '') {
+										setCustomSlippage('');
+										setSlippage(DEFAULT_SLIPPAGE);
+									} else if (numValue === 0) {
+										setCustomSlippage(value);
+										setSlippage(MIN_SLIPPAGE);
+									} else {
+										const clampedNum = clamp(numValue, MIN_SLIPPAGE, MAX_SLIPPAGE);
+										setSlippage(clampedNum);
+										setCustomSlippage(`${clampedNum}`);
 									}
```

Comment:
The logic here makes the input value be between `MIN_SLIPPAGE` and `MAX_SLIPPAGE` while also making the input clearable.